### DISCUSSION TITLE
`ui.icon_table_cell_type()` with `link = True`

### DISCUSSION THREAD

**Discussion by azim-b** [h2oai/wave#1062] [Ideas] [answered]
With a `ui.table()` could you make a column of icons that are "clickable", i.e. `link=True`?

I tried the following, which doesn't work:
```
ui.table_column(name='action', label='Action', cell_type=ui.icon_table_cell_type(color='red'), link=True)
```
![image](https://user-images.githubusercontent.com/64787868/136613058-b5689a2f-1def-4ad4-b676-e96d5bf10181.png)

**Comment by mturoci** [comment, marked as answer]
Hi @azim-b!

Icons are not currently clickable. We intend to provide `commands` for each table row in future, see https://github.com/h2oai/wave/issues/897.

**Reply by azim-b** [reply]
Thanks @mturoci 

I like the idea with #897. Since it may be a `Command`, we could also set an icon which is great.

But, in a use case where only a single action is required, having a clickable icon / link could be useful. What do you think?

**Reply by mturoci** [reply]
>But, in a use case where only a single action is required, having a clickable icon / link could be useful.

Agree, but it is more like a future nice to have so we will go with commands first probably as a universal solution to this.
